config/redirects

```diff
@@ -5,5 +5,43 @@ define: versions v1.1 v1.2 v1.3 v1.4 v1.5 v1.6 v1.7 v1.8 v1.9 v1.10 v1.11 v1.12
 symlink: upcoming -> master
```

Comment:
S: I can't comment directly on line 4, but I'd take out all the versions before v1.16

Reply:
good idea